docs/changelog.rst

```diff
@@ -69,6 +69,8 @@ Added
 
   The logo also comes in a light version, for dark backgrounds. See :ref:`brand` for details.
 
+* Vue streams: streams load more content on scroll (`#346 <https://github.com/jaywink/socialhome/pull/346>`_)
```

Comment:
No need for changelog entries for Vue stuff since it's not a feature yet - we'll just have one changelog entry when released 👍